binaries/apiserver/main.go

```diff
@@ -71,15 +72,15 @@ func main() {
 				sh.endpoint: sh.handler,
 			}
 		},
-		func(fileStore *bundlestore.FileStore, stat stats.StatsReceiver, tmp *temp.TempDir) (*StoreAndHandler, error) {
+		func(fileStore *bundlestore.FileStore, stat stats.StatsReceiver, upTimeReportIntv stats.UpTimeReportIntvl, tmp *temp.TempDir) (*StoreAndHandler, error) {
 			cfg := &bundlestore.GroupcacheConfig{
 				Name:         "apiserver",
 				Memory_bytes: 2 * 1024 * 1024 * 1024, //2GB
 				AddrSelf:     *httpAddr,
 				Endpoint:     "/groupcache",
 				Cluster:      createCluster(),
 			}
-			store, handler, err := bundlestore.MakeGroupcacheStore(fileStore, cfg, stat)
+			store, handler, err := bundlestore.MakeGroupcacheStore(fileStore, cfg, stat, upTimeReportIntv)
```

Comment:
And if a component needs to take multiple different intervals? I'd strongly prefer to keep stats confined as much as possible and given the *current* code I think it's ok to move it to stat and perhaps make the name more unique.